app/org/sagebionetworks/bridge/services/AuthenticationService.java

```diff
@@ -36,20 +47,27 @@
 
 @Component("authenticationService")
 public class AuthenticationService {
-    
+
     private final Logger logger = LoggerFactory.getLogger(AuthenticationService.class);
     
+    private static final EmailVerificationValidator EMAIL_VERIFICATION_VALIDATOR = new EmailVerificationValidator();
+    private static final SignInValidator EMAIL_SIGNIN_REQUEST_VALIDATOR = new SignInValidator(EMAIL_REQUEST);
+    private static final SignInValidator PASSWORD_SIGNIN_VALIDATOR = new SignInValidator(PASSWORD);
+    private static final SignInValidator EMAIL_SIGNIN_VALIDATOR = new SignInValidator(EMAIL);
+    private static final EmailValidator EMAIL_VALIDATOR = new EmailValidator();
```

Comment:
Minor: Instead of creating these as constants in AuthenticationService, would be better to create a singleton in the class itself? For example, EmailVerificationValidator.INSTANCE or SignInValidator.EMAIL_REQUEST_VALIDATOR.